js/runtime.js

```diff
@@ -1,17 +1,36 @@
 (function(global){
     'use strict';
 
+    // Dependencies: ctx, canvas, Event, runtime, sound, soundEffect,
     // canvas/stage stuff
-    var canvas, ctx;
+    var _canvas, _ctx;
+    function canvas(){
+        if (!_canvas){
+            if (dom.find){
+                _canvas = dom.find('wb-playground > canvas');
+            }
+            if (!_canvas){
+                _canvas = document.createElement('canvas');
+                _canvas.setAttribute('width', '200');
+                _canvas.setAttribute('height', '200');
```

Comment:
Maybe there should be a warning that the canvas element that is used throughout is not attached to the DOM when it is created here. It might be confusing otherwise.